packages/documentation/copy/en/handbook-v2/Object Types.md

```diff
@@ -154,26 +154,25 @@ function paintShape({ shape, xPos = 0, yPos = 0 }: PaintOptions) {
 
 Here we used [a destructuring pattern](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) for `paintShape`'s parameter, and provided [default values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Default_values) for `xPos` and `yPos`.
 Now `xPos` and `yPos` are both definitely present within the body of `paintShape`, but optional for any callers to `paintShape`.
+Using [mapping modifiers](/docs/handbook/2/mapped-types.html#mapping-modifiers), you can remove `optional` attributes.
```

Comment:
The rest of the change seems fine (provided code blocks render inside quote blocks), but this line seems out of place; mapping modifiers aren't mentioned anywhere in this section, right? Was this sentence (previously at the bottom) actually intended as an intro? Or maybe, it should just be removed?